src/userList3.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { GetClientObjects } from "./util/userDataListUtil";
+
+const UserList3 = ({ userDataList }) => {
+  const clientObjectList = GetClientObjects(userDataList);
+
+  return <UserListTree clientObjectList={clientObjectList} />;
+};
+
+const UserListTree = ({ clientObjectList }) => (
+  <div>
```

Comment:
Often we don't really use an actual element in components, and instead opt for a fragment (`<> </>`), unless we need to use a css class or add an accessibility property
```suggestion
  <>
```

Reply:
Sounds good